src/MKRTHERM.cpp

```diff
@@ -96,7 +96,18 @@ float THERMClass::readTemperature()
   // multiply for the LSB value
   celsius = rawword * 0.25f;
 
-  return celsius;
+  if(units=FAHRENHEIT)
+  {
+      return (celsius * 9.0 / 5.0) + 32.0;
+  }
+  else if(units=KELVIN)
+  {
+      return (celsius + 273.15);
+  }
+  else if(units=CELSIUS)
+  {
+      return celsius;
```

Comment:
```suggestion
  if (units == FAHRENHEIT) {
    return (celsius * 9.0 / 5.0) + 32.0;
  } else if(units == KELVIN) {
    return (celsius + 273.15);
  } else if(units == CELSIUS) {
    return celsius;
```
- Fix bug caused by the use of assignment operator instead of comparison operator.
- Make code formatting consistent with the rest of the file.